bool gripper_on
bool enable_motors